src/Atmos/Model/AtmosModel.jl

```diff
@@ -173,11 +169,7 @@ function AtmosModel{FT}(
     param_set::AbstractParameterSet;
     orientation::O = SphericalOrientation(),
     ref_state::RS = HydrostaticState(
-        LinearTemperatureProfile(
-            FT(200),
-            FT(280),
-            FT(grav(param_set)) / FT(cp_d(param_set)),
-        ),
+        DecayingTemperatureProfile(FT(200), FT(60), FT(8 * 10^3)),
```

Comment:
Don't you want 
```suggestion
        DecayingTemperatureProfile(FT(290), FT(60), FT(8 * 10^3)),
```
here, i.e., first argument is surface temperature?